Source/SIMPLib/Messages/AbstractStatusMessage.h

```diff
@@ -50,33 +49,25 @@ class SIMPLib_EXPORT AbstractStatusMessage : public AbstractMessage
   SIMPL_SHARED_POINTERS(AbstractStatusMessage)
   SIMPL_TYPE_MACRO(AbstractStatusMessage)
 
-  virtual ~AbstractStatusMessage()
-  {
-  }
+  ~AbstractStatusMessage() override;
 
   /**
    * @brief This method creates and returns a string for status messages
    */
-  virtual QString generateMessageString() const = 0;
+  QString generateMessageString() const override = 0;
```

Comment:
It would probably be best to just remove cases where we override a pure virtual method with a pure virtual method.  There is nothing being added, and it seems to imply that the base actually has an implementation that needs to be ignored or overridden in derived classes.